### PR TITLE
Websocket messageing

### PR DESCRIPTION
**Pull request by KhalilovJ**

### CODE REVIEW

File: src/main/java/az/evilcastle/crossingtherubicon/model/dto/websocket/messaging/WebsocketMessageParent.java [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+import lombok.AllArgsConstructor;
+import lombok.Builder;
+import lombok.Data;
+import lombok.NoArgsConstructor;
+
+@Builder
+@Data
+@NoArgsConstructor
+@AllArgsConstructor
+@JsonIgnoreProperties(ignoreUnknown = true)
```
Comment (malishov) [inline src/main/java/az/evilcastle/crossingtherubicon/model/dto/websocket/messaging/WebsocketMessageParent.java:14]: Why do we use it?

File: src/main/java/az/evilcastle/crossingtherubicon/model/dto/websocket/messaging/WebsocketMessageParent.java [hunk 2]
```diff
@@ -0,0 +1,18 @@
...
+public class WebsocketMessageParent {
```
Comment (malishov) [inline src/main/java/az/evilcastle/crossingtherubicon/model/dto/websocket/messaging/WebsocketMessageParent.java:15]: I suggest you use records for DTOs

File: src/main/java/az/evilcastle/crossingtherubicon/service/WebSocketHandlerService.java [hunk 3]
```diff
@@ -13,7 +16,9 @@
 
 @Component
 @Slf4j
+@RequiredArgsConstructor
 public class WebSocketHandlerService extends TextWebSocketHandler implements SubProtocolCapable {
+    private final ObjectMapper objectMapper = new ObjectMapper();
     @Override
```
Comment (malishov) [inline src/main/java/az/evilcastle/crossingtherubicon/service/WebSocketHandlerService.java:22]: Clean Code. Formatting. Vertical Density.

Add separating line

File: src/main/java/az/evilcastle/crossingtherubicon/service/WebSocketHandlerService.java [hunk 4]
```diff
@@ -32,6 +37,18 @@ public void afterConnectionClosed(WebSocketSession session, CloseStatus status)
...
     public void handleTextMessage(WebSocketSession session, TextMessage message) throws Exception {
         log.info("WebSocketSession message received; sessionId: {}; Message: {}", session.getId(), message.getPayload());
+        var requestMessage = objectMapper.readValue(message.getPayload(), WebsocketMessageParent.class);
```
Comment (malishov) [inline src/main/java/az/evilcastle/crossingtherubicon/service/WebSocketHandlerService.java:40]: Add separating line

File: src/main/java/az/evilcastle/crossingtherubicon/service/WebSocketHandlerService.java [hunk 5]
```diff
@@ -32,6 +37,18 @@ public void afterConnectionClosed(WebSocketSession session, CloseStatus status)
...
+        requestMessage.setWebsocketId(session.getId());
+        handleMessage(requestMessage);
```
Comment (malishov) [inline src/main/java/az/evilcastle/crossingtherubicon/service/WebSocketHandlerService.java:42]: log. end

File: src/main/java/az/evilcastle/crossingtherubicon/service/WebSocketHandlerService.java [hunk 6]
```diff
@@ -32,6 +37,18 @@ public void afterConnectionClosed(WebSocketSession session, CloseStatus status)
     @Override
     public void handleTextMessage(WebSocketSession session, TextMessage message) throws Exception {
         log.info("WebSocketSession message received; sessionId: {}; Message: {}", session.getId(), message.getPayload());
```
Comment (malishov) [inline src/main/java/az/evilcastle/crossingtherubicon/service/WebSocketHandlerService.java:39]: log start and use format : ActionLog.methodName.start

Comment (malishov) [inline src/main/java/az/evilcastle/crossingtherubicon/service/WebSocketHandlerService.java:39]: info or debug lvl ?

File: src/main/java/az/evilcastle/crossingtherubicon/service/WebSocketHandlerService.java [hunk 7]
```diff
@@ -1,5 +1,8 @@
 package az.evilcastle.crossingtherubicon.service;
 
+import az.evilcastle.crossingtherubicon.model.dto.websocket.messaging.WebsocketMessageParent;
```
Comment (malishov) [inline src/main/java/az/evilcastle/crossingtherubicon/service/WebSocketHandlerService.java:3]: From Codestyle. Separate our packages from 3rd party ( add separating line)

File: src/main/java/az/evilcastle/crossingtherubicon/model/dto/websocket/messaging/WebsocketMessageParent.java (hunk 1 above)
Comment (KhalilovJ) [reply src/main/java/az/evilcastle/crossingtherubicon/model/dto/websocket/messaging/WebsocketMessageParent.java:14]: Потому что jackson не маппит обьекты если в них лишние поля, а нам надо смаппить обьект в родителя изначально

File: src/main/java/az/evilcastle/crossingtherubicon/service/WebSocketHandlerService.java (hunk 5 above)
Comment (KhalilovJ) [reply src/main/java/az/evilcastle/crossingtherubicon/service/WebSocketHandlerService.java:42]: It's insignificant

File: src/main/java/az/evilcastle/crossingtherubicon/service/WebSocketHandlerService.java (hunk 6 above)
Comment (KhalilovJ) [reply src/main/java/az/evilcastle/crossingtherubicon/service/WebSocketHandlerService.java:39]: logging in receiver method is insignificant

File: src/main/java/az/evilcastle/crossingtherubicon/model/dto/websocket/messaging/WebsocketMessageParent.java (hunk 2 above)
Comment (KhalilovJ) [reply src/main/java/az/evilcastle/crossingtherubicon/model/dto/websocket/messaging/WebsocketMessageParent.java:15]: Which records?

Comment (malishov) [reply src/main/java/az/evilcastle/crossingtherubicon/model/dto/websocket/messaging/WebsocketMessageParent.java:15]: https://blog.payara.fish/expressive-rest-resources-with-java-records-and-jakarta-rest#:~:text=Java%20Records%20As%20DTOs,ensuring%20data%20integrity%20across%20layers.

**Pull request merged**